ext/mysqli/tests/mysqli_constants.phpt

```diff
@@ -10,6 +10,7 @@ mysqli.allow_local_infile=1
 $IS_MYSQLND = stristr(mysqli_get_client_info(), "mysqlnd");
 $constants = get_defined_constants(true);
 sort($constants);
+$IS_MARIADB = $constant('MYSQLI_IS_MARIADB');
```

Comment:
What does this line do? Should this really be a variable or did you mean a function?

Reply:
typo :)